bot/code_review_bot/workflow.py

```diff
@@ -294,9 +295,15 @@ def build_task(self, task_id, task_status):
             return CoverityTask(task_id, task_status)
         elif name == "source-test-infer-infer":
             return InferTask(task_id, task_status)
-        elif name.startswith("source-test-"):
-            logger.error(
-                "Unsupported source-test task: will need a local implementation"
-            )
         else:
-            raise Exception("Unsupported task {}".format(name))
+
+            # If default artifacts available > boom
```

Comment:
We should file an issue (if we don't already have one) to move all analyzers to the common format, so we can cleanup a bit the code.

Reply:
It probably would be better to file a Bugzilla meta bug ?

Reply:
Yep! And an issue in this repo to get rid of the code when that lands in mozilla-central

Reply:
Filed [Bug 1590669](https://bugzilla.mozilla.org/show_bug.cgi?id=1590669) & #180